### PR TITLE
Add Default

### PR DESCRIPTION
**Pull request by Nirma**
## Project URL
https://github.com/Nirma/Default

## Description
add the library Default to the list
 
## Why it should be included to `awesome-ios` (optional)
This is an _awesome_ alternative to using `UserDefaults`

## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 / tvOS 10 or later
- [x] Supports Swift 3 or later
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -431,6 +431,8 @@ Awesome-iOS is an amazing list for people who need a certain feature on their ap
 * [WCDB](https://github.com/Tencent/wcdb) - WCDB is an efficient, complete, easy-to-use mobile database framework for iOS, macOS.
 * [StorageKit](https://github.com/StorageKit/StorageKit) - Your Data Storage Troubleshooter 🛠
 * [UserDefaults](https://github.com/nmdias/DefaultsKit) - Simple, Strongly Typed UserDefaults for iOS, macOS and tvOS :large_orange_diamond:
+* [Default](https://github.com/Nirma/Default) - Modern interface to UserDefaults + Codable support
+ :large_orange_diamond:
```
Comment (dkhamsing) [inline README.md]: hi @Nirma can u put the 🔶 right after "Codable support"

**Comment by Nirma** [comment]
@dkhamsing oops, sorry about that! Ok I fixed it, how does it look now?

**Comment by dkhamsing** [comment]
Looks good, thx!

**Pull request merged**